Tasks/CopyFilesOverSSH/task.json

```diff
@@ -81,6 +81,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "failOnEmptySource",
+            "type": "boolean",
+            "label": "Fail when the Source folder is empty",
```

Comment:
We also want to fail when the source folder is not empty, but no files match to be copied.  For that case, I suggest renaming the label from:
`Fail when the Source folder is empty`
To:
`Fail when no files are found to copy`